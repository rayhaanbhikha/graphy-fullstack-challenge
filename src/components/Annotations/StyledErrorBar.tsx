import styled from "styled-components";

export const StyledErrorBar = styled.div`
  background: #d8d8d838;
  max-width: 300px;
  text-align: center;
  color: #ff4b4b;
  padding: 5px;
  border-radius: 5px;
`