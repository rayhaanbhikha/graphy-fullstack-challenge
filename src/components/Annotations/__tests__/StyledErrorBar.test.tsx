import React from 'react';
import { render } from '@testing-library/react'
import { StyledErrorBar } from '../StyledErrorBar';
import 'jest-styled-components';

describe('StyledErrorBar', () => {

  it('should have the default styles', () => {
    const { asFragment } = render(<StyledErrorBar />);
    expect(asFragment()).toMatchSnapshot();
  })
});