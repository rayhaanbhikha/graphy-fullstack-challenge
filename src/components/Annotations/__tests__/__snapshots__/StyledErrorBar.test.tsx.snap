// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledErrorBar should have the default styles 1`] = `
<DocumentFragment>
  .c0 {
  background: #f9f9f9;
  max-width: 300px;
  text-align: center;
  color: #ff4b4b;
  padding: 5px;
  border-radius: 5px;
  z-index: 1000;
  position: fixed;
}

<div
    class="c0"
  />
</DocumentFragment>
`;
